tools/SetupFlow/DevHome.SetupFlow/Strings/en-us/Resources.resw

```diff
@@ -1764,12 +1768,59 @@
     <value>There was an error retreiving the adaptive card session from the extension</value>
     <comment>Error text display when we are unable to retrieve the adaptive card information from an extension</comment>
   </data>
+  <data name="EnvironmentCreationReviewTabTitle" xml:space="preserve">
+    <value>Environment</value>
+    <comment>Title for create environment review tab</comment>
+  </data>
+  <data name="EnvironmentCreationReviewExpanderDescription" xml:space="preserve">
+    <value>Your environment's details</value>
+    <comment>Title for create environment review tab</comment>
+  </data>
   <data name="EnvironmentCreationReviewPageTitle" xml:space="preserve">
-    <value>Create Environment</value>
+    <value>Review your environment</value>
     <comment>Title for create environment review page</comment>
   </data>
   <data name="CreateEnvironmentButtonText" xml:space="preserve">
     <value>Create Environment</value>
-    <comment>Text for button that starts creating a new virtual machine or cloud pc for the user</comment>
+  </data>
+  <data name="EnvironmentCreationError" xml:space="preserve">
+    <value>There was an error starting the creation operation</value>
+    <comment>Text to tell the user that we couldn't start the operation to create their local or cloud virtual environment</comment>
+  </data>
+  <data name="EnvironmentCreationOperationInitializationFinished" xml:space="preserve">
+    <value>The operation to create your environment has started successfully</value>
+    <comment>Text to tell the user that we were able to start the operation that create their local or cloud virtual machine successfully</comment>
+  </data>
+  <data name="StartingEnvironmentCreation" xml:space="preserve">
+    <value>Starting the create environment operation</value>
+    <comment>Text to tell the user that the operation to create their local or cloud virtual environment has started</comment>
+  </data>
+  <data name="EnvironmentCreationFailedToGetProviderInformation" xml:space="preserve">
+    <value>We timed out waiting for the extension to provide us with information to create your environment</value>

```

Comment:
Should probably be 'Dev Home' instead of we.

Reply:
We use 'we' in other Windows UI places like in the Windows Settings app -> startup apps page when we couldn't find an app, for example. We're going over the strings with the content writer before build so this will probably get updated any way